src/main/java/com/fulcrumgenomics/util/PickLongIndices.scala

```diff
@@ -70,7 +69,10 @@ case class IndexMetric(index: String,
                        min_mismatches: Int,
                        indices_at_min_mismatches: Int,
                        gc: Double,
-                       min_delta_g: Option[Double]) extends Metric {
+                       longest_homopolymer: Int,
+                       worst_structure_seq: Option[String],
+                       worst_structure: Option[String],
```

Comment:
I don't want `worst_structure_structure`, but I kind of do.  It wasn't obvious what should be stored in this string until I went and found where it was set.  Could you add in some scaladoc for this metric?